Sources/ScrollViewProxy/ScrollViewProxy.swift

```diff
@@ -110,7 +109,14 @@ public struct ScrollViewProxy<ID: Hashable> {
     fileprivate var coordinator = Coordinator<ID>()
     fileprivate var space: UUID = UUID()
 
-    fileprivate init() {}
+    fileprivate init() {
+        coordinator.delegate.onScroll = { scrollView in
+            self.offset = CGPoint(
+                x: scrollView.contentOffset.x + scrollView.adjustedContentInset.horizontal,
+                y: scrollView.contentOffset.y + scrollView.adjustedContentInset.vertical
+            )
+        }
+    }
```

Comment:
The compiler doesn't allow us to capture `self` in the onScroll closure. 

I tried using bindings and setting `onScroll` inside `Coordinator.scrollView.didSet`, but I couldn't find a way to pass the offset to a superview using bindings or preferences.

Reply:
Ah I was working on my old laptop and I thought this was gonna work. What is the warning you get?

Reply:
The error for the above change is "Escaping closure captures mutating 'self' parameter". 

I thought there might have been a solution via this [SO answer](https://stackoverflow.com/a/60252772).

But it seems we can't apply the same solution because `ScrollViewProxy` isn't a class.

```swift
    fileprivate init() {
        weak var weakSelf: Self? // 'weak' may only be applied to class and class-bound protocol types, not 'ScrollViewProxy<ID>'
        coordinator.delegate.onScroll = { scrollView in
            weakSelf?.offset = CGPoint(
                x: scrollView.contentOffset.x + scrollView.adjustedContentInset.horizontal,
                y: scrollView.contentOffset.y + scrollView.adjustedContentInset.vertical
            )
        }
        weakSelf = self 
    }
```